start.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 /opt/envsubst < /envsubst_template.json > /conf/runtime.json
-exec java -Xms512m -Xmx3072m -jar /git-bridge.jar /conf/runtime.json
+exec java -XX:+UseContainerSupport -XX:MaxRAMPercentage=90.0 -jar /git-bridge.jar /conf/runtime.json
```

Comment:
I'm wondering if this might be too high, based on other overheads than the heap. Could we start at `75`?
```suggestion
exec java -XX:+UseContainerSupport -XX:MaxRAMPercentage=75.0 -jar /git-bridge.jar /conf/runtime.json
```